.changeset/fuzzy-insects-warn.md

```diff
@@ -0,0 +1,7 @@
+---
+"@frontity/types": minor
+---
+
+Add types for frontity options (`state.frontity.options.frontity_name` & `state.frontity.options.frontity_source_auth`)
```

Comment:
Just a reminder!

When you have changed the frontity options to camelCase, you will have to update this text.

Reply:
good catch, thanks 🙂